scripts/data_analysis/constants/common.py

```diff
@@ -5,11 +5,46 @@
 
 import pandas as pd
 
-URL = 'https://api-dev.prison.service.justice.gov.uk/v3/api-docs'
+DEFAULT_URL = 'https://api-dev.prison.service.justice.gov.uk/v3/api-docs'
+URLS = [
```

Comment:
Think this variable rename from URL to URLS has broken `discover_schema_endpoints.py`

Reply:
Good spot, must have missed that one.